docs/configuration/tempo-config.md

```diff
@@ -126,11 +126,17 @@ automatic_logging:
 # Supported receivers: otlp, jaeger, kafka, opencensus and zipkin.
 receivers: <receivers>
 
-# A list of prometheus scrape configs.  Targets discovered through these scrape
-# configs have their __address__ matched against the ip on incoming spans. If a
-# match is found then relabeling rules are applied.
-scrape_configs:
-  - [<scrape_config>]
+prom_sd:
+  # A list of prometheus scrape configs.  Targets discovered through these scrape
+  # configs have their __address__ matched against the ip on incoming spans. If a
+  # match is found then relabeling rules are applied.
+  scrape_configs:
+    - [<scrape_config>]
```

Comment:
imo I'd prefer not nesting scrape_configs so it feels similar to the hierarchy used by logs / metrics. Can `sd_operation_type` be another top-level key maybe?

Reply:
Yeah, that's the other option I considered. I chose this approach for consistency with other processors' configs, but I'm not a huge fan of another breaking change. I'll change it 👍

Reply:
Changed in ea2ccc0b279d79aad4855954debe2c0d2c8d5821